modules/payloads/stagers/python/reverse_tcp_ssl.rb

```diff
@@ -1,5 +1,5 @@
 ##
-# This module requires Metasploit: http//metasploit.com/download
+# This module requires Metasploit: https://www.metasploit.com/download
```

Comment:
`rake spec` will do that for you. You'll need to set up a test database. See `config/database.yml.example` for more details.